workflows/phaseimpute/tests/main.nf.test.snap

```diff
@@ -31,6 +31,27 @@
         },
         "timestamp": "2025-06-18T18:23:01.818626445"
     },
+    "Check test_minimac4": {
+        "content": [
+            [
+                "imputation/minimac4/concat/NA12878.minimac4.vcf.gz",
+                "imputation/minimac4/concat/NA12878.minimac4.vcf.gz.csi",
+                "imputation/minimac4/concat/NA19401.minimac4.vcf.gz",
+                "imputation/minimac4/concat/NA19401.minimac4.vcf.gz.csi",
+                "imputation/minimac4/concat/NA20359.minimac4.vcf.gz",
+                "imputation/minimac4/concat/NA20359.minimac4.vcf.gz.csi"
```

Comment:
This did seems to work at the workflow level.
The issue might be on the meta map used in the subworkflow test.